app/views/questionnaire.py

```diff
@@ -293,11 +369,18 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
         return redirect(invalid_location.url(metadata))
 
 
+def store_submitted_responses():
+    if g.schema_json.get('view_submitted_responses'):
+        return g.schema_json['view_submitted_responses'].get('enabled', False)
+
+    return False
+
+
 def is_survey_completed(full_routing_path):
     completed_blocks = get_completed_blocks(current_user)
 
     for location in full_routing_path:
-        if location.block_id in ['thank-you', 'summary', 'confirmation']:
+        if location.block_id in ['thank-you', 'summary', 'confirmation', 'view-submission']:
```

Comment:
I think these need to be changed to look at the Block type. But that may be another PR?